pkg/cmd/server/api/types.go

```diff
@@ -980,3 +983,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
ClusterResourceOverride works fine for the controller -
ClusterResourceOverridePolicy is probably appropriate for the config
*if* we will never have other resource policy in the same place.